packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# 0. Switch to the directory of the script, as gradle commands won't otherwise work
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+cd $DIR
+
+# 1. Check if the script is run from the gutenberg-mobile submodule
+PARENT_GIT_REPO_PATH=$(git rev-parse --show-superproject-working-tree)
+if [[ -z $PARENT_GIT_REPO_PATH ||
+    $(git -C $PARENT_GIT_REPO_PATH config --get remote.origin.url) != *"/gutenberg-mobile.git" ]]; then
+    echo "This script needs to be run from the 'gutenberg-mobile' submodule."
```

Comment:
The message tricked me up, it made me think I should `cd` into the `gutenberg-mobile` submodule of this project, while the requirement is that the script is called when this project is a submodule of `gutenberg-mobile`.

Would you consider this wording?

```suggestion
    echo "Could not find `gutenberg-mobile` parent folder. This script can run only when this project is contained within `gutenberg-mobile` as a Git submodule."
```

Reply:
Reading it again, I agree that it is not a good error message. I don't want to say `gutenberg-mobile` folder, because the folder can have a different name, so I came up with the below message in 
https://github.com/WordPress/gutenberg/pull/29488/commits/e58d4d22c3c81efc46845d116b2077344bc5f80f, what do you think?

> This script can only be used if the 'gutenberg' project is a submodule of the 'gutenberg-mobile' project

It looks like I forgot to put `.` at the end 🤦‍♂️ We'll remove this message when we move the script to `gutenberg-mobile` in https://github.com/wordpress-mobile/gutenberg-mobile/issues/3238, so I don't want to add another commit if the message is good enough.

Reply:
Sounds good!